build.gradle

```diff
@@ -75,6 +76,22 @@ jacocoTestCoverageVerification {
 
 check.dependsOn jacocoTestCoverageVerification
 
+checkstyle {
+    toolVersion = 9.0
+    showViolations = true
+    ignoreFailures = true
+    config = rootProject.resources.text.fromFile('config/checkstyle/checkstyle.xml');
```

Comment:
is this required? won't it search for the exact same file by default?

Reply:
Okay